monitoring/prober/rid/v2/test_isa_simple.py

```diff
@@ -80,7 +80,8 @@ def test_get_isa_by_id(ids, session_ridv2):
 
 @default_scope(SCOPE_SP)
 def test_update_isa(ids, session_ridv2):
-  resp = session_ridv2.get('{}/{}'.format(ISA_PATH, ids(ISA_TYPE)))
+  resp = session_ridv2.get('{}/{}'.format(ISA_PATH, ids(ISA_TYPE)), scope=SCOPE_DP)
```

Comment:
Same as test_is_expiry.py -- this should stay as SCOPE_SP after `updated.yaml` is updated

Reply:
Now fixed